src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -75,10 +87,14 @@ export const ClickableStyle = React.forwardRef(
       // Variants
       variant === 'primary' && styles['clickable-style--primary'],
       variant === 'secondary' && styles['clickable-style--secondary'],
-      variant === 'tertiary' && styles['clickable-style--tertiary'],
       variant === 'icon' && styles['clickable-style--icon'],
       variant === 'link' && styles['clickable-style--link'],
-      variant === 'destructive' && styles['clickable-style--destructive'],
+      // Colors
+      status === 'brand' && styles['clickable-style--brand'],
```

Comment:
sorry if you already had this convo -- for default values for `variant` and `status`, could we put those styles directly in the main `.clickable-style` class & cut down the className size for most use cases?

to be fair we're probably not being consistent about this across the board, so could file a ticket & come back to it